projects/managed-hubs/sales.md

```diff
@@ -1,96 +1,99 @@
-# Leads, sales, and hubs process
+# Sales and invoicing process
 
-2i2c pursues new collaborations and contracts via following-up on leads for those who may be interested in 2i2c services.
-This page has information to help guide this process.
-
-:::{admonition} Out of date!
-:class: warning
-The information on this page is out of date, and will be updated once we complete our onboarding with {term}`CS&S`.
-:::
+This section has information about pursuing and creating new contracts for 2i2c services.
 
 ## Location of sales assets and documents
 
 Here are some locations of common documents used in the sales process (other than the repositories described below).
 
 [The Managed Hub Service folder](https://drive.google.com/drive/folders/1QuvUvwFxPAxw-bJ6_zjgwKXPurHC6UfW?usp=sharing)
-: Contains all folders and information that cover the Managed Hub Service, including sales and contracting.
+: Contains all folders and information that cover the Managed Hub Service, including assets for creating new contracts and following leads.
 
-[The leads folder](https://drive.google.com/drive/folders/1msuG0xWPbawMwesD8LZxdMClP-GVpgT2?usp=sharing)
-: Contains a folder for each lead that 2i2c is currently engaging. This is where we store client-specific information, such as price quotes or contracts that have been sent to them.
+[The leads and partnerships AirTable](https://airtable.com/appDUEjjcSqHfilFD/tblxLUQ3ElIaaQuM8/viwXstuM3K48smZXW)
+: An AirTable with the pipeline for potential communities we will serve.
+  This table is the source of truth about who we are talking to about serving hub infrastructure, and who we have an active service with.
 
-[The sales assets folder](https://drive.google.com/drive/folders/1aMZILBmFSTYBSB9EwyV5wRpcuprM06dJ?usp=sharing)
+[The partnerships and leads assets folder](https://drive.google.com/drive/folders/1aMZILBmFSTYBSB9EwyV5wRpcuprM06dJ?usp=sharing)
 : Contains all templates, email copy, stationery, etc that is useful in 2i2c sales.
 
-## The Leads to Sales workflow
-
-### Overview
+## Who participates in this process?
 
-We use `Monday.com` to track our leads
-: [**This Monday Board**](https://icsi-company.monday.com/boards/1140216585) keeps track of all leads in the sales process. Below is an overview of each step in the process.
+Anybody at 2i2c that is interested in growing our service to new communities is welcome to participate in the process below.
```

Comment:
@choldgraf how we would manage the private information if everyone is welcome to participate in the process?

Reply:
by that I mean anybody on the @2i2c-org/tech-team or the @2i2c-org/steering-council